app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter_for_post
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- **[OPTIONAL]** Just like we have in the user modal, `after_save :update_comments_counter`